esmvalcore/dataset.py

```diff
@@ -848,3 +867,19 @@ def _update_timerange(self):
         check.valid_time_selection(timerange)
 
         self.set_facet('timerange', timerange)
+
+    def _get_temporary_fixed_file_dir(self) -> Path:
+        """Create and return new temporary directory for storing fixed files.
+
+        Returns
+        -------
+        Path
+            Path to new temporary directory.
+
+        """
+        fixed_file_dir = self.session._fixed_file_dir
+        fixed_file_dir.mkdir(parents=True, exist_ok=True)
+        prefix = self._get_joined_summary_facets('_', join_lists=True)
+        if prefix != '':
```

Comment:
This never happens, it's impossible to load a dataset without specifying at the very least project, mip, variable, dataset.